### PR TITLE
i18n | New translations for auth and auth-front

### PR DESCRIPTION
**Pull request by bkoval**
Fresh translations for the auth project.

@Warkot @armonr Please take a look :)

### CODE REVIEW

**Comment by Warkot** [comment]
I'm not an expert on Chinese, but I trust it, as it was downloaded from Crowdin. Therefore it LGTM. :+1:

File: front/locales/zh/auth-front.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "password_exceeds_max_length":"密碼不能超過50個字元",
     "password_equal_name":"用戶名和密碼不匹配",
     "user_already_linked":"這個Wikia帳戶已經連接到Facebook",
-    "fb_account_already_linked":"這個Facebook帳戶已經連接到另一個Wikia用戶帳戶",
+    "fb_account_already_linked":"這個Facebook帳戶已經連接到另一個Wikia帳戶",
```
Comment (armonr) [inline front/locales/zh/auth-front.json]: Just for fun, in crowdin looks like this. `这个Facebook帐户已经连接到另一个Wikia用户帐户`. But if that's what gets downloaded, that is what it is and LGTM too :)

**Comment by Warkot** [comment]
And now for something completely different…
Why most of the keys in `auth-front.json` are written with **snake_case** instead of **kebab-case**?

File: front/locales/zh/auth-front.json (hunk 1 above)
Comment (Warkot) [inline front/locales/zh/auth-front.json]: Indeed, in Crowdin Chinese simplified is:
```
这个Facebook帐户已经连接到另一个Wikia用户帐户
```
However, Chinese traditional is:
```
這個Facebook帳戶已經連接到另一個Wikia帳戶
```

Comment (armonr) [inline front/locales/zh/auth-front.json]: Interesting. It makes me want to simply learn traditional Chinese

**Comment by bkoval** [comment]
Talked to @warkot offline about i18n keys, basically they're compliant with errors returned by Helios and snake-casing is making them easier to map.

**Pull request merged**